packages/cosmjs/src/cosmoskit/chains.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * A pre-configured chain object for Sei for use with Cosmos Kit.
+ * @category Cosmos Kit
+ */
+export const SEI_BASE_COSMOS_KIT_CHAIN = {
```

Comment:
can we pull this from the registry instead? makes it easier to update gas price and other values if needed

Reply:
not sure what the best practice is here - maybe we can even set up a separate package so we can update the info there instead of having to do a version bump on cosmjs?